### DISCUSSION TITLE
Score calculation tool - Original: Change selected item when viewing in tour mode

### DISCUSSION THREAD

**Discussion by dtra** [MKT-Playbook/Hub#10] [Feature requests]
It would be nice if you could switch a driver/kart/glider (mainly driver) to see how it affects the score estimate. You already show the alternatives below each selected item, it’d be nice if there was some functionality to show the score estimate using different items from that alternative list.

**Comment by TitusPosiadly** [comment]
Ah yeah, good idea. I would have to make quite a lot of changes to the app structure for that though. Maybe a dedicated score calculation tool would be an alternative, where you could select a track and compare two different load-outs? Otherwise, I will keep this in mind whenever I do some major changes to the tour page.

**Reply by dtra82** [reply]
Yeah that would be useful, as long as the loadouts could be easily changed.
